pulp_2to3_migration/app/plugin/deb/migrator.py

```diff
@@ -0,0 +1,50 @@
+import asyncio
+
+from pulp_2to3_migration.app.plugin.api import (
+    Pulp2to3PluginMigrator,
+    ContentMigrationFirstStage,
+    DeclarativeContentMigration,
+)
+
+from pulp_deb.app import models as pulp3_models
+
+from . import pulp2_models
+from . import pulp_2to3_models
+from .repository import DebImporter, DebDistributor
+
+
+class DebMigrator(Pulp2to3PluginMigrator):
+    """
+    An entry point for migration the Pulp 2 Debian plugin to Pulp 3.
+
+    Attributes:
+      Check parent class.
+    """
+    pulp2_plugin = 'deb'
+    pulp2_content_models = {
+        'deb': pulp2_models.DebPackage,
+    }
+    pulp2_collection = 'units_deb'
+    pulp3_plugin = 'pulp_deb'
+    pulp3_repository = pulp3_models.DebRepository
+    content_models = {
+        'deb': pulp_2to3_models.Pulp2DebPackage,
+    }
+    importer_migrators = {
+        'deb_importer': DebImporter,
+    }
+    distributor_migrators = {
+        'deb_distributor': DebDistributor,
+    }
+
+    @classmethod
+    def migrate_content_to_pulp3(cls):
+        """
+        Migrate pre-migrated Pulp 2 Debian content.
+        """
+        first_stage = ContentMigrationFirstStage(cls)
+        dm = DeclarativeContentMigration(first_stage=first_stage)
+
+        loop = asyncio.get_event_loop()
+        loop.run_until_complete(dm.create())
+        loop.close()
```

Comment:
Delete this line in accordance with https://github.com/pulp/pulp-2to3-migration/pull/151/files

Reply:
Will do, I generally try to check all incoming commits whenever I rebase to a newer `master` :wink:.

Reply:
Should be done as part of a rebase to current master.